src/mds/Server.cc

```diff
@@ -2529,7 +2529,6 @@ void Server::dispatch_client_request(MDRequestRef& mdr)
   if (is_full) {
     CInode *cur = try_get_auth_inode(mdr, req->get_filepath().get_ino());
     if (!cur) {
-      respond_to_request(mdr, -EINVAL);
```

Comment:
Let's add a comment that the request is already responded to!

Reply:
(great catch @lxbsz !)

Reply:
Sorry, I missed this comment, I will update it. Thanks @batrick